packages/sdk/src/space.test.ts

```diff
@@ -150,8 +148,18 @@ describe('spaceTests', () => {
             mimetype: 'image/png',
             filename: 'bob-1.png',
         })
-
-        await bobsClient.setSpaceImage(spaceId, mediaStreamId, image)
+        const { key, iv } = await deriveKeyAndIV(nanoid(128)) // if in browser please use window.crypto.subtle.generateKey
+        const chunkedMediaInfo = {
+            info: image,
+            streamId: mediaStreamId,
+            encryption: {
+                case: 'aesgcm',
+                value: { secretKey: key, iv },
+            },
+            thumbnail: undefined,
+        } satisfies PlainMessage<ChunkedMedia>
+
+        await bobsClient.setSpaceImage(spaceId, chunkedMediaInfo)
```

Comment:
concern about having the caller derive the key+iv. That such a critical part of the derived-key protocol. If the caller gets it wrong, then a lot of flow breaks.

Simplifying the ChunkedMedia encryption back to only `aesgcm` makes a lot of sense. But the key+iv derivation should be inside `setSpaceImage` to avoid a lot of subsequent decryption issues.